### ISSUE TITLE
Webfont Italic with wrong reference/name

### ISSUE THREAD

**Issue by guilhermeasouza** [uswds/public-sans#91]
Hi, thanks for keeping the font updated.

Installing Public Sans today I came across a problem. Apparently one or more fonts are overwriting each other possibly because they have the same references / names. Is this happening?

Example: Italic / SemiBold Italic

Thank you for your support.

**Comment by thisisdano** [comment]
Hm! I haven't seen anything like this but I can look at it. Does this happen when installing the OTF versions of the fonts locally?

**Comment by guilhermeasouza** [comment]
> Hm! I haven't seen anything like this but I can look at it. Does this happen when installing the OTF versions of the fonts locally?

Exactly. It actually happened first with the OTF file. So I uninstalled and installed the Web / TTF version. The same problem occurred, which was the absence of the Regular font. So I uninstalled, rebooted and installed just Regular, and it worked. So I installed one by one.

When I tried to install Italic and then SemiBold, only one of them is displayed. Strange that the last time I tried the SemiBold Italic font was available but with the weight of Regular Italic. That's why I commented on the possibility of having something with the Name.

One important thing, I'm using the Sketch program on MAC OS to use Public Sans. In the Font Book it appears normally.

For the work I'm doing I ended up not using Italic, so I continued the work. But I leave the feedback, I can not tell if the problem is in the font or the Sketch.
